src/core/thread/fiber.d

```diff
@@ -929,9 +930,9 @@ private:
     }
     do
     {
-        // adjust alloc size to a multiple of PAGESIZE
-        sz += PAGESIZE - 1;
-        sz -= sz % PAGESIZE;
+        // adjust alloc size to a multiple of pageSize
+        sz += pageSize - 1;
+        sz -= sz % pageSize;
```

Comment:
```suggestion
        sz -= sz & (pageSize -1);
```
This becomes a lot slower when `pageSize` isn't a compile time constant. Since further down you allow the assumption that `pageSize` is a power of 2 I suggest doing so here too.